File: src/net/merayen/elastic/uinodes/list/poly_1/Info.java
package net.merayen.elastic.uinodes.list.poly_1;

import net.merayen.elastic.uinodes.BaseInfo;

public class Info implements BaseInfo {
	@Override
	public String getName() {return "Poly";}

	@Override
	public String getDescription() {return "Polyphonic group";}

	@Override
	public String[] getCategories() {return new String[]{"Tool"};}
}

File: src/main/kotlin/net/merayen/elastic/uinodes/list/ramp_1/Info.kt
package net.merayen.elastic.uinodes.list.ramp_1

import net.merayen.elastic.uinodes.BaseInfo

class Info : BaseInfo {
	override fun getName() = "Ramp"
	override fun getDescription() = "Converts level to non-linear levels"
	override fun getCategories() = arrayOf("Convert")
}